source/deployment/admin-roles.rst

```diff
@@ -45,51 +51,119 @@ Each role has a set of default permissions, and when a user is assigned a role,
 Assigning Admin Roles
 ---------------------
 
-System Admins can assign roles and modify privileges using the mmctl tool. This can be done either locally or remotely.
+System Admins can assign roles using the System Console or the mmctl tool. This can be done either locally or remotely.
+
+1. Go to **System Console > User Management > System Roles (Beta) > Assigned People**. 
 
 The format of the mmctl command is:
 
 ``mmctl permissions role assign [role_name] [username...]``
 
 **To grant the System Manager role to a single user called Bob Smith:**
 
+1. Go to **System Console > User Management > System Roles (Beta)** and select the **System Manager** role.
+
+2. Under **Assigned People**, click **Add People**.
+
+3. Search for and select ``Bob Smith``, and then click **Add** to grant the System Manager role to that user.
+
+The format of the mmctl command is:
+
 ``mmctl permissions role assign system_manager bob-smith``
 
 **To grant the User Manager role to two users, Bob Smith and Sue Clark:**
 
+1. Go to **System Console > User Management > System Roles (Beta)** and select the **User Manager** role.
+
+2. Under **Assigned People**, click **Add People**.
+
+3. Search for and select ``Bob Smith`` and ``Sue Clark``, and then click **Add** to grant the User Manager role to those users.
+
+The format of the mmctl command is:
+
 ``mmctl permissions role assign system_user_manager bob-smith sue-clark``
 
 **To grant the Read Only Admin role to two users, Bob Smith and Sue Clark:**
 
+1. Go to **System Console > User Management > System Roles (Beta)** and select the **Viewer** role.
+
+2. Under **Assigned People**, click **Add People**.
+
+3. Search for and select ``Bob Smith`` and ``Sue Clark``, and then click **Add** to grant the Viewer role to those users.
```

Comment:
```suggestion
3. Search for and select **Bob Smith** and **Sue Clark**, then select **Add** to grant the Viewer role to those users.
```

Reply:
In this line, based on our style guide, Viewer should be *Viewer* as it's a role/user. I just realized that which is why it's not formatted in that way. Thoughts on changing the formatting in this doc? Examples of this formatting are in the installation guide (e.g., https://docs.mattermost.com/install/install-ubuntu-2004.html), so it is used in the docs albeit inconsistently.